bundle.py

```diff
@@ -30,18 +30,20 @@
 HERE = os.path.abspath(os.path.dirname(__file__))
```

Comment:
I guess changes in this file can be ignored for the purposes of this PR.

Reply:
What does this mean exactly?